src/firefly/js/drawingLayers/HiPSMOC.js

```diff
@@ -454,25 +459,27 @@ function updateMocData(inDrawLayer, plotId) {
      }
 }
 
-function getAutoDrawStyle(plot, color) {
+function getAutoDrawStyle(plot, color,autoUsesOnlyOutline=false) {
     const [r,g,b]= toRGBA(color);
     const fov= getFoV(plot);
     if (!fov) return {style:Style.FILL, color:toRGBAString([r,g,b,1])};
-    if (fov<26) return {style:Style.DESTINATION_OUTLINE, color:toRGBAString([r,g,b,1])};
-
-    let alpha= .7;
-    if (fov>170) alpha= .7;
-    else if (fov>150) alpha= .68;
-    else if (fov>130) alpha= .65;
-    else if (fov>110) alpha= .58;
-    else if (fov>90) alpha= .55;
-    else if (fov>70) alpha= .45;
-    else if (fov>50) alpha= .35;
-    else if (fov>40) alpha= .25;
-    else if (fov>30) alpha= .20;
+    if (autoUsesOnlyOutline || fov<30) return {style:Style.DESTINATION_OUTLINE, color:toRGBAString([r,g,b,1])};
+    const alpha= getAlpha(fov);
     return {style:Style.FILL, color: toRGBAString([r,g,b,alpha])};
 }
 
+function getAlpha(fov) {
+    const MIN_FOV   = 30;
+    const MAX_FOV   = 170;
+    const MIN_ALPHA = 0.2;
+    const MAX_ALPHA = 0.7;
```

Comment:
something worth trying, I have a hunch this will give the cleanest transition from transparently filled to only outlined:

```suggestion
    const MIN_FOV   = 25;
    const MAX_FOV   = 170;
    const MIN_ALPHA = 0.01;
    const MAX_ALPHA = 0.7;
```